src/main/java/com/ververica/flink/table/gateway/operation/InsertOperation.java

```diff
@@ -186,6 +187,7 @@ protected void cancelJobInternal() {
 		// blocking deployment
 		try {
 			JobClient jobClient = deployer.deploy().get();
+			clusterDescriptorAdapter.setClusterIdValue(configuration.getString(YarnConfigOptions.APPLICATION_ID, ""));
```

Comment:
This class should not see `YarnConfigOptions.APPLICATION_ID`

Reply:
Got it